tests/Doctrine/Tests/ORM/EntityManagerTest.php

```diff
@@ -174,31 +174,33 @@ public function testAffectedByErrorIfClosedException($methodName)
         $this->_em->$methodName(new \stdClass());
     }
 
+    public function dataToBeReturnedByTransactional()
+    {
+        return array(
+            array(null),
+            array(false),
+            array('foo'),
+        );
```

Comment:
I'm using the long-array-syntax here because it's used in other data providers. I'm guessing it will be changed to the short-syntax later in a batch.